test/D05/TC01.cs

```diff
@@ -0,0 +1,50 @@
+﻿using aoc.D05;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace aoc.test.D05
+{
+	[TestClass]
+	public class TC01
+	{
+		BoardingPass[] passes = Parser.Parse("./D05/input.txt");
+
+		[TestMethod]
+		public void InitialI()
```

Comment:
I would do something similar to a `Theory` in xunit. I expect this testing framework can do similar.

Reply:
I looked up. Its clunky n bad in mstest. I do not like xUnit coz its slow in debugging. xUnit is better  architecture for sure. For thsi simple case, seems like an overkill. Wont do.